Bio/PDB/ResidueDepth.py

```diff
@@ -487,18 +484,14 @@ def get_surface(model, PDB_TO_XYZR=None, MSMS="msms"):
 
 
 def min_dist(coord, surface):
-    """
-    Return minimum distance between coord
-    and surface.
-    """
+    """Return minimum distance between coord and surface."""
     d = surface - coord
     d2 = numpy.sum(d * d, 1)
     return numpy.sqrt(min(d2))
 
 
 def residue_depth(residue, surface):
-    """
-    Return average distance to surface for all
+    """Return average distance to surface for all
```

Comment:
Instead insert a one line summary like ``Residue depth as average depth of all its atoms.``